pkg/shipyard/io_test.go

```diff
@@ -116,6 +116,8 @@ var _ = Describe("Io", func() {
       LABEL com.github.30x.shipyard.revision=v1.0
 
       CMD ["npm", "start"]
+
```

Comment:
necessary to pass diff tests. The template leaves empty space when there are no environment variables for the image